lib/utilities.js

```diff
@@ -41,11 +41,11 @@ const alias = (
   return wrap;
 };
 
-const safe = (
-  // Make function raise-safe
-  fn // function
-  // Returns: function, wrapped with try/catch interception
-) => (...args) => {
+// Make function raise-safe
+//   fn - function
+// Returns: function, wrapped with try/catch interception
+//   ...args - array, arguments to be passed to wrapped function
```

Comment:
I think descriding arguments we do not need `...`
So not `...args - array` but `args - array`

Reply:
Fix this in all places

Reply:
Furthermore, when generating docs, we will get
```md
### Function description
`fn(args)`
```
instead of
```md
### Function description
`fn(...args)`
```